src/main/java/frc/trigon/robot/subsystems/turret/toohardturret/ToohardTurretConstants.java

```diff
@@ -0,0 +1,83 @@
+package frc.trigon.robot.subsystems.turret.toohardturret;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.*;
+
+public class ToohardTurretConstants {
+    static final boolean FOC_ENABLED = true;
+    private static final int
+            MOTOR_ID = 0,
+            ENCODER_ID = 0;
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final double ENCODER_OFFSET = 0;
+    private static final SensorDirectionValue ENCODER_DIRECTION_VALUE = SensorDirectionValue.Clockwise_Positive;
+    private static final AbsoluteSensorRangeValue ENCODER_SENSOR_RANGE_VALUE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final FeedbackSensorSourceValue ENCODER_SENSOR_SOURCE_VALUE = FeedbackSensorSourceValue.RemoteCANcoder;
+    private static final double
+            P = 0,
+            I = 0,
+            D = 0;
+    private static final double
+            KS = 0,
+            KG = 0,
+            KV = 0,
+            KA = 0;
+    static final TalonFX MOTOR = new TalonFX(MOTOR_ID);
+    static final CANcoder ENCODER = new CANcoder(ENCODER_ID);
+
+    static final StatusSignal<Double>
+            ENCODER_POSITION_SIGNAL_DEGREES = ENCODER.getPosition(),
+            ENCODER_VELOCITY_SIGNAL_DEGREES_PER_SECOND = ENCODER.getVelocity();
```

Comment:
Remove units from the name, this is a status signal. Check what are the units given to you!